daprdocs/content/en/reference/api/bindings_api.md

```diff
@@ -37,6 +37,42 @@ If running on kubernetes apply the component to your cluster.
 
 > **Note:** In production never place passwords or secrets within Dapr component files. For information on securely storing and retrieving secrets using secret stores refer to [Setup Secret Store]({{< ref setup-secret-store >}})
 
+### Binding direction (optional)
+
+There are scenarios where it would be useful to provide additional information to Dapr to indicate the direction supported by the Binding Component. 
```

Comment:
```suggestion
In some scenarios, it would be useful to provide additional information to Dapr to indicate the direction supported by the binding component. 
```